lib/p5.sound.js

```diff
@@ -1,4 +1,4 @@
-/*! p5.sound.js v0.3.3 2017-06-06 */
```

Comment:
For the next PR, let's hold off on adding the lib to commits—it'll make it easier to review because we can just focus on the `src/` and `test/` files.